### PR TITLE
chore: enhance ESLint setup

### PR DESCRIPTION
**Pull request by tujoworker**
- Add tree new rules, two regarding comments and one for block if statements.
- Remove ESLint config in Portal and use it from Eufemia
- Change code regarding the new setup + rules

### CODE REVIEW

**Review by langz — APPROVED** [review]
Nice Nice, I like rules that enforces consistency👊

**Review by joakbjerk — APPROVED** [review]

**Comment by tujoworker** [comment]
@langz and @joakbjerk I have added an [extra commit](https://github.com/dnbexperience/eufemia/pull/2131/commits/84989a42f458a5ea2202d49b8b0917f84ed43909) with some docs about it. Can you check if it looks good and if something is missing?

**Review by joakbjerk — APPROVED** [review]
Looks good on my machine 👍

File: packages/dnb-design-system-portal/src/docs/contribute/style-guides/coding.mdx [hunk 1]
```diff
@@ -11,14 +11,9 @@ The current set of main code principles within JS, CSS, Typography and testing a
...
 
-- Use [React Testing Library](https://testing-library.com) when possible, even Enzyme is still used in existing components.
 - Use [Volta](https://volta.sh/) for [Node.js](https://nodejs.org/) version handling.
-- Use plugins in your favorite code editor for [ESLint](https://eslint.org/docs/user-guide/integrations) and [Prettier](https://prettier.io/docs/en/editors.html), which formats the code for you.
-
-Other helpful tools:
-
-- [Axe Devtools extension](https://www.deque.com/axe/devtools/)
-- [Lighthouse extension](https://developers.google.com/web/tools/lighthouse#devtools)
+- Use [ESLint](https://eslint.org/docs/user-guide/integrations) and [Prettier](https://prettier.io/docs/en/editors.html) Plugins in your favorite code editor to show related issues inline.
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/contribute/style-guides/coding.mdx:15]:
```suggestion
- Use [ESLint](https://eslint.org/docs/user-guide/integrations) and [Prettier](https://prettier.io/docs/en/editors.html) plugins in your favorite code editor to show related issues inline.
```

**Pull request merged**